packages/core/content-releases/admin/src/index.ts

```diff
@@ -1,13 +1,36 @@
 import { prefixPluginTranslations } from '@strapi/helper-plugin';
+import { PaperPlane } from '@strapi/icons';
+
+import { pluginId } from './pluginId';
+
+import type { Plugin } from '@strapi/types';
 
 // eslint-disable-next-line import/no-default-export
```

Comment:
Why change from 
```js
export default {
  register() {}
  ....
}
```

to:

```
const admin = { ... }

export default admin
```
?

I prefer the first way but if you have a reason is fine 🤔, also, if you keep it like that I guess you can remove this line because is not exporting here

Reply:
Same, I followed what was made in the email plugin which is in TS https://github.com/strapi/strapi/blob/main/packages/core/email/admin/src/index.ts